src/js/utils/common.js

```diff
@@ -87,20 +87,29 @@ export function getTextWidth(text) {
     return Math.ceil(context.measureText(text).width);
 }
 
-export function sortElement(element, keepDown = "DISABLED") {
-    const list = document.querySelector(element);
-    [...list.children]
-        .sort((a, b) => {
-            if (a.innerText === keepDown) {
-                return 1;
-            } else if (b.innerText === keepDown) {
-                return -1;
-            } else {
-                return a.innerText > b.innerText ? 1 : -1;
-            }
-        })
-        .forEach(node => list.appendChild(node));
-}
+/**
+ * Returns jquery sorted option list with optional value staying on top of the list.
+ *
+ * @param {string} optional value staying on top of the list.
+ * @return {object} sorted option list.
+ */
+
+$.fn.sortSelect = function(text = "") {
+    const op = this.children("option");
+
+    op.sort((a, b) => {
+        console.log(a.value, a.text);
+        if (a.text === text) {
+            return -1;
+        }
+        if (b.text === text) {
+            return 1;
+        }
+        return a.text.localeCompare(b.text, window.navigator.language, { ignorePunctiaton: true });
```

Comment:
It seems the `localeCompare` does not complain about typos :)

```suggestion
        return a.text.localeCompare(b.text, window.navigator.language, { ignorePunctuation: true });
```

Reply:
... and it seems to work, strange ...

Reply:
I suppose simply ignores the parameter, and not ignores de punctuation.

Reply:
https://reference.wolfram.com/language/ref/IgnorePunctuation.html :)